tests/function_corpus.py

```diff
@@ -13,9 +13,7 @@ class TestFunc:
     static_argnames: set = dataclasses.field(default_factory=lambda: set())
 
 def shape_strategy():
-    # Generate reasonable dimensions for tensor shapes
-    sizes = st.integers(min_value=1, max_value=5)
-    return st.lists(sizes, min_size=0, max_size=4).map(tuple)
+    return st.just((2,))
```

Comment:
For early testing, this is an ok strategy, but its better to modify your test locally as a non-hypothesis test first and then build up to a hypothesis test.